internal/auth0/prompt/resource_screen_render.go

```diff
@@ -0,0 +1,210 @@
+package prompt
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/auth0/go-auth0"
+	"github.com/auth0/go-auth0/management"
+
+	"github.com/auth0/terraform-provider-auth0/internal/config"
+	internalError "github.com/auth0/terraform-provider-auth0/internal/error"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/structure"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+var (
```

Comment:
Since, most of the values declared of type slice, and `const` doesn't support to slice type.
So, for better readability.. I've declared all values in vars